spec/features/client/client/client_register_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+feature 'Visitor creates Account' do
+  scenario 'successfully' do
+    visit root_path
+    click_on 'Registrar'
+    fill_in 'CPF', with: '082.923.869-71'
+    fill_in 'Email', with: 'test@email.com'
+    fill_in 'Senha', with: '12345678'
+    fill_in 'Confirmar senha', with: '12345678'
+    click_on 'Registrar'
+
+    expect(Client.all.count).to eq(1)
```

Comment:
Essa linha não agrega muito ao teste porque a linha de baixo já serve como verificação mas daria pra simplificar.

```suggestion
    expect(Client.count).to eq(1)
```